css/explore.css

```diff
@@ -1,25 +1,17 @@
-.explore {
-  text-align: center;
-  background-color: #F2F2F2;
-  height: auto;
-  padding: 3rem;
-}
-.explore_text h1 {
-  font-size: 2.5rem;
-  margin: 0px;
-}
-.explore_text p {
-  font-size: 2rem;
-}
-.explore_button {
-  background: linear-gradient(#FF79DA, #9356DC);
-  color: white;
-  border-radius: 40px;
-  border: none;
-  font-size: 2rem;
-  height: 5rem;
-  padding: 0px 35px;
-  box-shadow: 2px 2px 5px grey;
-}
+/* Error: Undefined variable.
```

Comment:
so what do we do ? do you keep that like that even if it's a useless comment ?

Reply:
I'll make sure to delete this comment